.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -6,23 +6,41 @@
   data retrieval. 
 -->
 
-## Description
+## Related issues
 
-- Link to related issues: `#<issue number>`
-- Description of the problem the contribution solves or link to design issue
-  - [ ] Explanation of how existing TDS code falls short
-  - [ ] Explanation of backwards compatibility for users who are already on this feature
+See <!-- put issue number here, or delete this section -->
 
-## Considerations
+## Notes
```

Comment:
I feel that a "Description" section is more appropriate as an optional section rather than this notes list.